config/v1/types_cluster_version.go

```diff
@@ -780,11 +806,24 @@ type ConditionalUpdate struct {
 	// +required
 	Release Release `json:"release"`
 
+	// riskNames represents the set of the names of conditionalUpdateRisks in the status that are exposed to the release in this conditional update.
+	// A condition update is accepted only if each of its risk is either not applied to the cluster or considered acceptable by the cluster administrator.
+	// Entries must be unique and must not exceed 256 characters.
+	// riskNames must not contain more than 500 entries.
+	// +openshift:enable:FeatureGate=ClusterUpdateAcceptedRisks
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:items:MaxLength=256
+	// +kubebuilder:validation:MaxItems=500
+	// +listType=set
+	// +required
```

Comment:
To verify, this means there will _never_ be a conditional update where there are no risks? Every conditional update _must_ have at least one risk?